src/Moryx.AbstractionLayer/Activities/Activity.cs

```diff
@@ -48,6 +54,9 @@ public abstract class Activity : IActivity
         internal Activity()
         {
             Tracing = new Tracing();
+
+            var displayName = GetType().GetDisplayName();
+            Name = !string.IsNullOrEmpty(displayName) ? displayName : GetType().Name;
```

Comment:
We should invert this. The `!` is easily overlooked and requires an unnecessary CPU cycle.

Reply:
done